egobee_test.go

```diff
@@ -7,30 +7,32 @@ import (
 	"time"
 )
 
-type fakeTokenStore struct {
+type fakeTokenStorer struct {
 	access  string
 	refresh string
 	vf      time.Duration
 }
 
-func (s *fakeTokenStore) AccessToken() string {
-	return s.access
+func (s *fakeTokenStorer) GetAccessToken() (string, error) {
```

Comment:
To keep these function signatures simpler, it might be better to return empty strings instead of an empty string _and_ an error. If the disk is corrupted or the file disappears, the next attempt to `Update()` will give us a meaningful error and the `authorizingTransport` will report that on the next request.

Same with `ValidFor()`, which can return a 0 duration (ie, "not valid").